data-esnext.js

```diff
@@ -1680,17 +1688,20 @@ exports.tests = [
              return true;
            }
            var arr = new Constructor([1, 2, 3]);
```

Comment:
`new BigInt64Array([1, 2, 3]) -> TypeError`

Reply:
Fixed

Reply:
Let's fix this by adding a separate BigInt test rather than just not testing them at all?

Reply:
Please feel free to add it in a separate patch.  This one is already 10 times the work I originally intended to do.